apps/vx-mark/frontend/jest.config.js

```diff
@@ -12,6 +12,14 @@ module.exports = {
     '!src/index.tsx',
     '!src/contexts/ballot_context.ts',
   ],
+  coverageThreshold: {
+    global: {
+      statements: 96,
+      branches: 87,
+      functions: 96,
+      lines: 96,
+    },
+  },
```

Comment:
Dropping the test coverage for the either neither work, should go back to 100% once you remove it @jonahkagan and you can remove this.

Reply:
Will do